addon/mixins/debounced-response.js

```diff
@@ -1,16 +1,19 @@
 import Ember from 'ember';
 
 export default Ember.Mixin.create({
-  debounce: function (handler) {
-    return () => {
-      if (!this.isScheduled) {
-        this.isScheduled = true;
+  debounce(handler) {
+    const self = this;
```

Comment:
Arrow functions don't bind `arguments` so that's why the function on [line 7](https://github.com/dollarshaveclub/ember-responds-to/pull/15/files#diff-392c80d688c764e4d4f97a2a128b6bc8R7) can't be an arrow function. For that reason we need the reference to `self`.